units/XNB1102/XNB1102_script.lua

```diff
@@ -1,11 +1,11 @@
--- hydro power plant
-
 local NomadsEffectTemplate = import('/lua/nomadseffecttemplate.lua')
 local NEnergyCreationUnit = import('/lua/nomadsunits.lua').NEnergyCreationUnit
 local AddRapidRepair = import('/lua/nomadsutils.lua').AddRapidRepair
 
 NEnergyCreationUnit = AddRapidRepair(NEnergyCreationUnit)
 
+--- Tech 1 Hydrocarbon
```

Comment:
This name matches gpg/faf repo file descriptions and is more clear that it is the structure and not the deposit.
```suggestion
--- Tech 1 Hydrocarbon Power Plant
```